board/main.c

```diff
@@ -703,15 +703,24 @@ void tick_handler(void) {
         siren_countdown -= 1U;
       }
 
+      if (controls_allowed) {
+        controls_allowed_countdown = 30U;
+      } else if (controls_allowed_countdown > 0U) {
+        controls_allowed_countdown -= 1U;
+      } else {
```

Comment:
this empty else seems like a dumb rule, maybe we should consider removing it from the misra check?

Reply:
if we want to do this, let's do it in another PR